Distribution specific patch

restored tests commented out in CMakeLists.txt upstream

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c53252b..ee06015 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -176,8 +176,8 @@ install(FILES "${CMAKE_CURRENT_BINARY_DIR}/whisper.pc"
 #

 if (WHISPER_BUILD_TESTS AND NOT CMAKE_JS_VERSION)
-    #include(CTest)
-    #add_subdirectory(tests)
+    include(CTest)
+    add_subdirectory(tests)
 endif ()

 if (WHISPER_BUILD_EXAMPLES)
